src/Plugin/Block/EzContentUserShortcutBlock.php

```diff
@@ -70,9 +70,9 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function build() {
     $build = [];
-    // Getting current user role.
-    $currentUserRoles = $this->account->getRoles();
-    if ($this->account->hasPermission('ezcontent user based shortcut')) {
+    if ($this->account->hasPermission('ezcontent shortcut set creation') && $this->account->id() != '1') {
+      // Getting current user role.
+      $currentUserRoles = $this->account->getRoles();
       if (!in_array('administrator', $currentUserRoles)) {
```

Comment:
if $currentUserRoles is being used only here, then let's avoid it and use $this->account->getRoles() directly.